src-theme/src/routes/clickgui/Panel.svelte

```diff
@@ -22,6 +22,9 @@
     let moving = false;
     let prevX = 0;
     let prevY = 0;
+    let offsetX = 0;
+    let offsetY = 0;
+    const GRID_SIZE = 20;
```

Comment:
Personally, I do not believe that 20 would be a good match for the size of the grid as the panel width cannot divide into 20 and leave a number without a decimal, meaning that the right hand side of the panel will not align.

Perhaps 25 will be a better number.

Reply:
Due to it being reviewed etc i will leave it like this for now, its only a small change. When it is officially added to nexgen, i will do another pull request with this change and more features.

Reply:
> Due to it being reviewed etc i will leave it like this for now, its only a small change. When it is officially added to nexgen, i will do another pull request with this change and more features.

You don’t need to do that by the way, the review has only been requested for this pr and not specific code so you can continue adding to it before it gets reviewed.

Reply:
If you feel like you **need** to add more features, make it a draft pull request so the reviewers know not to review yet.

Reply:
Oh i see, thanks, im gonna change it now. This is the first time doing a pull request so its all new for me.